Exiled.Events/Config.cs

```diff
@@ -54,6 +54,12 @@ public sealed class Config : IConfig
         [Description("Indicates whether the blood can be spawned or not")]
         public bool CanSpawnBlood { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether keycard throw can affect basic doors.
+        /// </summary>
+        [Description("Indicates whether keycard throw can affect on basic doors(door with empty permissions)")]
```

Comment:
I'd change it to: _Indicates whether thrown keycards can affect doors that don't require any permissions_